ledger-core/v2/application/builtin/contract/testwallet/testwallet.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package testwallet
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/v2/logicrunner/builtin/foundation"
+)
+
+// Wallet - basic wallet contract.
+type Wallet struct {
+	foundation.BaseContract
```

Comment:
please add amount/sum as field and add some fixed amount (a constant, for example 1000) to every created wallet.

Reply:
I planned to add it in https://insolar.atlassian.net/browse/PLAT-205

Reply:
If you'll add amount field, pls, use a larger value. Smth like a couple of millions should do. It will allow us to run prolonged tests and not run out of money :)